src/MauiReactor/Integration/ComponentHost.cs

```diff
@@ -32,6 +32,17 @@ public ComponentHostNode(ComponentHost host)
 
         BindableObject? ITemplateHost.NativeElement => ContainerPage;
 
+        
+        Microsoft.Maui.Controls.Page? IVisualNode.GetContainerPage()
+        {
+            return ContainerPage;
+        }
+
+        IHostElement? IVisualNode.GetPageHost()

```

Comment:
I am not sure if there are any side effects by putting this here, from what I can tell it is used to commit animations. it is possible that this would fix an unknown bug around animations in ComponentHosts but I have not investigated to see if there is a bug there. I primarily need `GetContainerPage` but added this so that it is consistent across the different `IHostElement`'s.

Reply:
Yes, it's primarily used to request the first animation frame from the hosting element (in this case, the component host), which drives the animation timer. Without your fix, I verified that the animation won't work at all.